fellowship.js

```diff
@@ -17,21 +29,51 @@ var fellowship = {
     "The Shire", "Rivendell", "Mordor"
   ],
   makeMiddleEarth: function(lands) {
-    console.log(lands)
-    // create a section tag with an id of middle-earth
-    // add each land as an article tag
-    // inside each article tag include an h1 with the name of the land
-    // append middle-earth to your document body
+    console.log(lands);
+    var nodeTitle = document.createTextNode("We're making middle earth ya hobbits");
+    document.getElementById("middle_earth").appendChild(nodeTitle);
+    for(var i=0; i<fellowship.lands.length; i++){
+      var fellowshipLand = fellowship.lands[i];
+      var landsNode = document.createElement("h1");
+      var landsTextNode = document.createTextNode(fellowshipLand);
+      landsNode.appendChild(landsTextNode);
+      document.getElementById("middle_earth").appendChild(landsNode);
+    }
   },
   makeHobbits: function(hobbits) {
+    console.log(hobbits);
+    var hobbitTitleNode = document.createElement("ul");
+    var hobbitTitle = document.createTextNode("Callin all the hobbits");
+    hobbitTitleNode.appendChild(hobbitTitle);
+    document.getElementById("middle_earth").appendChild(hobbitTitleNode);
+    for(var i=0; i<fellowship.hobbits.length; i++){
```

Comment:
Because you're passing in hobbits as an argument, you can reference `hobbits` instead of `fellowship.hobbits`.